custom-connectors/inQubaJourney/apiDefinition.swagger.json

```diff
@@ -0,0 +1,301 @@
+                  "type": "string",
+                  "description": "Authorization token that will be used for all API interactions.",
+                  "title": "Authorization Token"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Acquire Access Token",
+        "description": "Authenticate with the inQuba API and receive and authorization token that will be used for all API interactions.",
+        "operationId": "AcquireAccessToken",
+        "parameters": [
+          {
+            "name": "TenantName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tenant Name",
+            "description": "Name of Tenant"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/x-www-form-urlencoded",
+            "description": "Request Content Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "HostURL",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Host of Token endpoint must be supplied without scheme name. Eg. api.inqubacx.com",
+            "x-ms-summary": "Host URL"
+          },
+          {
+            "name": "username",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Username",
+            "x-ms-summary": "Username"
+          },
+          {
+            "name": "password",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Account Password",
+            "x-ms-summary": "Password"
+          },
+          {
+            "name": "grant_type",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "default": "password",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "client_id",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client ID",
+            "x-ms-summary": "Client ID"
+          },
+          {
+            "name": "scope",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal",
+            "default": "openid cx_api"
+          },
+          {
+            "name": "client_secret",
+            "in": "formData",
+            "required": true,
+            "type": "string",
+            "description": "Client Secret",
+            "x-ms-summary": "Client Secret"
+          }
+        ]
+      }
+    },
+    "/cems/api/Events": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@prinayinq Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Applies the same comment all the empty schema definitions in the swagger.

Reply:
@sriyen-msft does this mean the API methods need to return a body instead of an empty 200 response?

Reply:
@prinayinq Yes, it needs to be returned something which can be flagged in the network logs

Reply:
schema definition has been updated. Thanks